{{cookiecutter.project_slug}}/_/deployment/python/deployment/application/base/cronjob/jobs/hello/patch.yaml

```diff
@@ -0,0 +1,6 @@
+- op: replace
+  path: /spec/schedule
+  value: "*/5 * * * *"
+- op: replace
+  path: /spec/jobTemplate/spec/template/spec/containers/0/args
+  value: ['echo', 'hello world']
```

Comment:
IMHO this will produce an error each time the job runs (because cron jobs are expected to send not output in case of success, IIRC).

Reply:
I didn't notice it giving any error messages.  That said, maybe it's a good thing for it to generate error messages because the user is supposed to replace it with a real job -- getting error messages would remind the user to update it or turn it off.